packages/fast-usdc/src/exos/liquidity-pool.js

```diff
@@ -48,6 +48,18 @@ const { add, isGTE, makeEmpty } = AmountMath;
  * }} RepayPaymentKWR
  */
 
+export const stateShape = harden({
```

Comment:
should this get used when making the exo?

Reply:
Does this limit upgrade options? I'm fuzzy on the current exo upgrade constraints.

Reply:
good catch. I'll include it. TMK it doesn't constrain upgrade additionally because a state schema is implied even if one is not passed in.